tests/test_hear_keyboard_interrupt.py

```diff
@@ -60,6 +60,12 @@ class TestHearKeyboardInterrupt(BasicUsageTestSuite):
 	"""
 	__module__ = """tests.test_hear_keyboard_interrupt"""
 
+	# Constants for test configuration
+	STARTUP_DELAY_SECONDS = 1  # Allow server to start
+	PROCESS_TIMEOUT_SECONDS = 5
+	EXPECTED_SIGINT_EXIT_CODE = 130
+	INVALID_ARGS_EXIT_CODE = 2
+
 	def test_hear_keyboard_interrupt(self):
 		"""Tests the special hear and stop test"""
```

Comment:
Consider this improvement:

```suggestion
	"""
	Test proper handling of keyboard interrupts (SIGINT).

	This test:
		1. Starts a multicast server with coverage tracking
		2. Waits for server initialization
		3. Sends a SIGINT signal to simulate Ctrl+C
		4. Verifies that the server exits with the expected status code

	Success criteria:
		- Server must exit with status code 130 (standard SIGINT exit code)
		- Server must not exit with status code 2 (invalid arguments)
	"""
```

Reply:
Future work will be tracked in #255 for this suggested change.